src/workerd/io/worker-entrypoint.c++

```diff
@@ -507,36 +507,34 @@ kj::Promise<WorkerInterface::CustomEvent::Result>
   return maybeAddGcPassForTest(context, kj::mv(promise));
 }
 
-template <typename T>
-kj::Promise<T> WorkerEntrypoint::maybeAddGcPassForTest(
-    IoContext& context, kj::Promise<T> promise) {
-#ifdef KJ_DEBUG
```

Comment:
Sorry, I may have misled you. I forgot that we were already skipping this in debug builds, and I think there was actually a good reason: It breaks load tests. I think we do want to go on skipping this in debug builds after all.

Reply:
I assume you mean skipping in release builds?